docs/configuration.md

```diff
@@ -278,6 +280,18 @@ The eight-character Playlist ID of the Search playlist that you want to use to e
 
 ---
 
+**features.favorites_list** (optional)
```

Comment:
I recommend we keep these the same format as search and recommendations (i.e. `favoritesPlaylist`.) If this isn't what it is on the backend, let's fix it there before any of this goes out.

Reply:
Good catch Danny! I am still confused about differences in case notation. I will fix it on Monday after coming back from PTO.